tasty-query/shared/src/main/scala/tastyquery/reader/classfiles/ClassfileReader.scala

```diff
@@ -84,14 +79,11 @@ final class ClassfileReader private () {
         throw ReadException(s"Expected unforced UTF8 constant at index $idx")
     }
 
-    /** returns a new IArray with the decoded bytes, mutates `bytes` in-place, so `bytes` should not be shared
-      * after passing to this method.
-      */
+    /** Returns a new IArray with the decoded bytes. */
     private def unsafeDecodeSigBytes(bytes: IArray[Byte]): IArray[Byte] =
```

Comment:
plus fix the callsite
```suggestion
    private def decodeSigBytes(bytes: IArray[Byte]): IArray[Byte] =
```